menpo/image/base.py

```diff
@@ -1788,19 +1788,35 @@ def zoom(self, scale, cval=0.0):
         t = scale_about_centre(self, 1.0 / scale)
         return self.warp_to_shape(self.shape, t, cval=cval)
 
-    def rotate_ccw_about_centre(self, theta, degrees=True, cval=0.0):
+    def rotate_ccw(self, theta, degrees=True, cval=0.0, retain_shape=False,
+                   rotate_landmarks=True):
         r"""
-        Return a rotation of this image clockwise about its centre.
+        Return a counter-clockwise rotation of this image.
```

Comment:
It's probably worth noting that this **is** around the image centre. Because technically a rotation would be around the origin and would rotate the image out of the frame.